goprocessor/src/sawtooth_siimplewallet/handlers/simple_wallet_struct.go

```diff
@@ -0,0 +1,33 @@
+package handlers
```

Comment:
Typo error in the directory name? goprocessor/src/**sawtooth_siimplewallet**/handlers/simple_wallet_struct.go